src/equations/ideal_glm_mhd_2d.jl

```diff
@@ -822,6 +999,19 @@ end
     return SVector(w1, w2, w3, w4, w5, w6, w7, w8, w9)
 end
 
+# Transformation from conservative variables u to d(p)/d(u)
+@inline function dpdu(u, equations::IdealGlmMhdEquations2D)
+    rho, rho_v1, rho_v2, rho_v3, rho_e, B1, B2, B3, psi = u
+
+    v1 = rho_v1 / rho
+    v2 = rho_v2 / rho
+    v3 = rho_v3 / rho
+    v_square = v1^2 + v2^2 + v3^2
+
+    return (equations.gamma - 1.0) *
+           SVector(0.5 * v_square, -v1, -v2, -v3, 1.0, -B1, -B2, -B3, -psi)
+end
+
```

Comment:
```suggestion
@inline function pressure(u, equations::IdealGlmMhdEquations2D, derivative::True)
    return dpdu(u, equations)
end

```
Probably, it would be best to add a general function to `equations.jl` instead of adding it for every equation, right? Does it contain the same parameter for every equation?

The same holds for the both entropy variable routines.

Reply:
Well, maybe we can remove dpdu and just keep `pressure(u, equations::IdealGlmMhdEquations2D, derivative::True)`?

Reply:
Yes, we can do that:+1:

Reply:
Done in baeaf029

Reply:
Could you please do the same for `CompressibleEulerEquations2D`?

Reply:
Done in 342bf608

Reply:
Thanks